frontend/src/layout/navigation/Breadcrumbs/breadcrumbsLogic.tsx

```diff
@@ -202,9 +159,7 @@ export const breadcrumbsLogic = kea<breadcrumbsLogicType>([
                         key: 'project',
                         name: featureFlags[FEATURE_FLAGS.ENVIRONMENTS] ? currentProject.name : currentTeam.name,
                         tag: featureFlags[FEATURE_FLAGS.ENVIRONMENTS] ? currentTeam.name : null,
-                        popover: {
-                            overlay: <ProjectSwitcherOverlay />,
-                        },
+                        is_popover_project: true,
```

Comment:
Let's keep the project breadcrumb conforming with the rest of breadcrumbs. In this variant, it's extremely special, which creates a few snags:

![2025-07-09 13 05 19](https://github.com/user-attachments/assets/f057ddfa-ad3e-4921-b979-319aa22915dc)

1. Breadcrumbs have a different left margin from the scene title
2. The font size is off
3. The hover styling is button-like, which is not the case for all other breadcrumbs